$NetBSD: patch-ag,v 1.14 2023/04/30 14:58:58 spz Exp $

don't set the pgp path here, always use $INN::Config::pgp

--- control/pgpverify.in.orig	2014-05-17 08:24:49.000000000 +0000
+++ control/pgpverify.in
@@ -139,7 +139,7 @@
 # Path to pgp binary; for PGP 5.0, set the path to the pgpv binary.  If
 # you have INN and the script is able to successfully include your
 # INN::Config module, the value of $INN::Config::pgp will override this.
-$pgp = '/usr/local/bin/pgp';
+# $pgp = '/usr/local/bin/pgp';
 
 # If you keep your keyring somewhere that is not the default used by pgp,
 # uncomment the next line and set appropriately.  If you have INN and the
